lib/src/dropdown/Dropdown.tsx

```diff
@@ -242,10 +242,7 @@ const calculateWidth = (margin, size) =>
     : sizes[size];
 
 const DropdownContainer = styled.div<{ margin: DropdownPropsType["margin"]; size: DropdownPropsType["size"] }>`
-  display: inline-block;
   width: ${(props) => calculateWidth(props.margin, props.size)};
-  text-overflow: ellipsis;
-  overflow: hidden;
```

Comment:
This removes a previously existing margin. Has it been validated by the design team?